internify/backend/routes/api/users.js

```diff
@@ -0,0 +1,72 @@
+var express = require("express");
+var router = express.Router();
+var UserData = require("../../models/UserData");
+
+//@route    GET api/users/getAll
+//@desc     Get All User Documents
+//@access   Public (?)
+
+router.get("/", function (req, res, next) {
+  UserData.find()
+    .then((users) => res.status(200).json(users))
+    .catch((err) => res.status(404).json({ success: false }));
+});
+
+//@route    GET api/users/getOne/:id
+//@desc     Get single User Document
+//@access   Public
+
+router.get("/getOne/:id", function (req, res, next) {
```

Comment:
followup: if the former is true, then we shouldn't need to specify `getOne` or `getMany`. Rather, since we are already specifying the `id` in the URI, it is assumed that we are then getting one back. We should change this route to `/users/:id`

Reply:
In the former GET request, we already are getting back many users when we didn't specify the `id`

Reply:
I agree with this, we should change the url, we can make the same change to the student routes too

Reply:
Ok so I'll change router.get("/getOne/:id") to router.get("/:id"). I can do this for both routes-> users.js and routes->jobs.js

Reply:
yes sir